test/Pict_contentassignment_tests.js

```diff
@@ -48,6 +48,64 @@ suite
 						fDone();
 					}
 				);
+
+				test
+				(
+					'Check an element for a class name, then add and remove classes',
+					(fDone) =>
+					{
+						let testPict = new libPict(_MockSettings);
+						let classMap = {};
+						// for 'hasClass'
+						testPict.ContentAssignment.customReadClassFunction = (pAddress, pClass) =>
+						{ 
+							if (pClass === 'mustard') 
+							{
+								return true;
+							} 
+							else 
+							{
+								return false;
+							}
+						}
```

Comment:
The preference in this repo seems to be to add this to the EnvironmentObject class and use that class in your tests

Reply:
Updated